### PR TITLE
Update proxy and api sha to tips of release-1.3

### PR DESCRIPTION
**Pull request by mandarjog**
1. Update proxy sha
2. Update api sha
3. Update stats tags to match new proxy
4. cherry-pick #15551 which updates bootstrap metadata to match new proxy.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -155,7 +155,7 @@ require (
 	gopkg.in/square/go-jose.v2 v2.3.1
 	gopkg.in/stack.v1 v1.7.0 // indirect
 	gopkg.in/yaml.v2 v2.2.2
-	istio.io/api v0.0.0-20190813012407-93fc23ae4a32
+	istio.io/api v0.0.0-20190820204432-483f2547d882
```
Comment (rlenglet) [inline go.mod]: This doesn't match the version in `istio/proxy` for the SHA you chose in `istio.deps`:
https://github.com/istio/proxy/blob/3245d3582d7e4937150b2c56120a556420721ac3/istio.deps
It should be `36b42252042c0e8b229a0d66059af184c3105f9d `.

Comment (rlenglet) [reply go.mod]: Discussed with @mandarjog. The API SHA in `istio/proxy:release-1.3` points to a commit of `istio/api` that is >1 month old. This needs to be updated first to use the same version as in `istio/istio:release-1.3`.

Comment (rlenglet) [reply go.mod]: https://github.com/istio/proxy/pull/2384

**Comment by mandarjog** [comment]
@kyessenov please create a new release of go-controlplane .
I need to include it in this commit. `AllowUnknownFields` has been removed in favor of
`allow_unknown_static_fields` and `reject_unknown_dynamic_fields`

File: istio.deps [hunk 2]
```diff
@@ -4,7 +4,7 @@
 		"name": "PROXY_REPO_SHA",
 		"repoName": "proxy",
 		"file": "",
-		"lastStableSHA": "49bf92867731f6c108581e55131083fe0b35687f"
+		"lastStableSHA": "3245d3582d7e4937150b2c56120a556420721ac3"
```
Comment (rlenglet) [inline istio.deps]: This should now be https://github.com/istio/proxy/commit/42eec5f2d1af7718491b98fa82afb14118d1fff3, from https://github.com/istio/proxy/pull/2384.

File: go.mod [hunk 3]
```diff
@@ -46,7 +47,8 @@ require (
 	github.com/duosecurity/duo_api_golang v0.0.0-20190308151101-6c680f768e74 // indirect
 	github.com/elazarl/go-bindata-assetfs v1.0.0 // indirect
 	github.com/emicklei/go-restful v2.9.3+incompatible
-	github.com/envoyproxy/go-control-plane v0.8.6
+	github.com/envoyproxy/go-control-plane v0.8.7-0.20190821215049-f062b07a671a
```
Comment (rlenglet) [inline go.mod]: LGTM. I checked that this points to a version of Envoy that is just a couple days older than the Envoy version used in istio/proxy.

**Pull request merged**